sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.net6.0.cs

```diff
@@ -723,6 +725,13 @@ public enum BlobGeoReplicationStatus
         Bootstrap = 1,
         Unavailable = 2,
     }
+    public partial class BlobGetPropertiesOptions
```

Comment:
I don't think we should create a separate Options bag class for this because it won't add value for customers.  They'll only see a single property and now have to write `new BlobGetPropertiesOptions { Conditions = foo }` instead of just `foo` to call the method.  It'd be better to create a new method overload taking a second parameter (with no defaults) and [EB(Never)] that overload.